src/test/java/org/jvnet/hudson/test/RealJenkinsRuleTest.java

```diff
@@ -95,6 +93,13 @@ private static void _smokes(JenkinsRule r) throws Throwable {
         assertEquals(rr.getUrl().toExternalForm(), rr.runRemotely(RealJenkinsRuleTest::_getJenkinsUrlFromRemote));
     }
 
+    @Test public void testJenkinsLocationConfiguration() throws Throwable {
+        rr.startJenkins();
+        String[] jenkinsLocations = rr.getHome().list(
+                (dir, name) -> name.equalsIgnoreCase("jenkins.model.JenkinsLocationConfiguration.xml"));
+        assertThat(jenkinsLocations, IsArrayWithSize.arrayWithSize(1));
```

Comment:
Use the `Matchers` import. Anyway I think you would want to assert the actual XML content.